Unreadable config file cases crash

Version: 0.1.5

An inacessable config file causes a crash:
```
time="2020-04-22T07:34:35Z" level=info msg="Reading configuration from [/etc/kube-vip/config.yaml]"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1277a9e]

goroutine 1 [running]:
github.com/plunder-app/kube-vip/cmd.glob..func6(0x21107c0, 0xc0001a5c00, 0x0, 0x2)
	/src/cmd/kube-vip-start.go:53 +0x44e
github.com/spf13/cobra.(*Command).execute(0x21107c0, 0xc0001a5be0, 0x2, 0x2, 0x21107c0, 0xc0001a5be0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:844 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0x2110a60, 0x43b80a, 0x20b7720, 0xc000058750)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:945 +0x317
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.6/command.go:885
github.com/plunder-app/kube-vip/cmd.Execute()
	/src/cmd/kube-vip.go:64 +0x31
main.main()
	/src/main.go:14 +0x6e
```
kube-vip should have a proper error handling in that case and present a warning.

--- a/cmd/kube-vip-start.go
+++ b/cmd/kube-vip-start.go
@@ -50,10 +50,10 @@ var kubeVipStart = &cobra.Command{
 
 		if configPath != "" {
 			c, err := kubevip.OpenConfig(configPath)
-			startConfig = *c
 			if err != nil {
 				log.Fatalf("%v", err)
 			}
+			startConfig = *c
 		}
 
 		// parse environment variables, these will overwrite anything loaded or flags

